### ISSUE TITLE
Impossible save required field

### ISSUE THREAD

**Issue by illycz** [TimOetting/kirby-builder#179]
Kirby Builder 2.0.15
Kirby 3.4.3

Plainkit installation within simple blueprint:
```
title: Default Page
preset: page
fields:
  text:
    label: Text
    type: textarea
    size: large
  mybuilder:
    label: Page Builder
    type: builder
    fieldsets:
      quote:
        label: Quote
        fields:
          citation:
            label: Citation
            type: text
            required: true
```

I can't save page with populated field:
![image](https://user-images.githubusercontent.com/2109984/93920180-195bba00-fd0f-11ea-8db7-a3242833c906.png)

Error form console:
![image](https://user-images.githubusercontent.com/2109984/93920229-2d072080-fd0f-11ea-8154-acb147d9532d.png)

Any suggestion?
My client can't update pages within Kirby builder.

Thanks!

**Comment by illycz** [comment]
When I downgrade to Kirby 3.4.2 everything working...

**Comment by chrfickinger** [comment]
I have a similar problem with kirby builder and the latest kirby 3.4.3.
It seems the strange behavior only appears for pulic and unlisted pages on my project. Switching to draft mode solves the problem.
Could you please check if this is working for your project as well!

There is definitly an issue with required fields (without when condion). Required fields (with when condition) works as expected.

**Comment by findthebug** [comment]
we do have the same issue.

**Comment by illycz** [comment]
> Could you please check if this is working for your project as well!

Yes, It's possible save draft page, but when I try switch status to public, I get the same error...

**Comment by illycz** [comment]
Maybe it's related to new functionality "Required fields and when conditions": https://github.com/getkirby/kirby/releases/tag/3.4.3

**Comment by chrfickinger** [comment]
I have the same assumption. Hopefully the hint with the draft mode helps to debug the error faster.

**Comment by maxhoffmann** [comment]
we have also run into this on multiple projects. so far the only solution is to keep the older Kirby version

**Comment by chrfickinger** [comment]
Probably the plugin will not be fixed anymore, because there are plans to integrate it into the core of Kirby soon.
See: https://kirby.nolt.io/

**Comment by bavarianbytes** [comment]
Same problem here

**Comment by andreasotto** [comment]
any news on this?

**Comment by luglio7** [comment]
Same error

**Comment by dinko** [comment]
Same here & following…

**Comment by luglio7** [comment]
Not even a reply in a month, this is very allarming.

**Comment by findthebug** [comment]
100% sure this will not be fixed. the plugin will be replaced by the native one, bastian is working on. but for me a clear comment from @TimOetting would just be decent.

**Comment by chrfickinger** [comment]
I think if he had the time, he would also comment on it. As far as I know, he helped the guys at Kirby get the plugin to the core, and most of the problems are already fixed there. They've done a great job so far.

**Comment by findthebug** [comment]
it's not about doing a great job, they do. it's about clarity. we have customers and project they depend on this. i we just get an clear answer to all the questions here it would be great.

**Comment by chrfickinger** [comment]
Yes, you are right. I recommend to stay on kirby 3.4.2 for the moment. This bug came up just with 3.4.3.
If Kirby 3.5 is relased (what should be pretty soon), you can use the new builder. It's already in use on my project and i had nothing to adjust.

**Comment by TimOetting** [comment]
Finally fixed with version 2.0.16

**Comment by kkauper** [comment]
Hi, I have another error now. So far saving the required fields is working, but when I am using a Structure Field inside a Builder Field -- with required fields -- the same error as before occurs. 
So far, I have not tested going back to Kirby 3.4.2 and check if it is working here. Would be nice, if this -- hopefully small enhancement -- could be added to the Builder Field.
